tests/gtest/avifgainmaptest.cc

```diff
@@ -44,7 +44,7 @@ void CheckGainMapMetadataMatches(const avifGainMapMetadata& lhs,
   }
 }
 
-avifGainMapMetadata GetTestGainMapMetadata(bool base_rendition_is_hdr) {
+avifGainMapMetadata GetTestGainMapMetadata() {
```

Comment:
IMPORTANT: I wonder if we should keep the `base_rendition_is_hdr` boolean parameter and let it control `metadata.baseHdrHeadroomN/D` and `metadata.alternateHdrHeadroomN/D`.

Right now `baseHdrHeadroom` is hardcoded to be 0 (0/1) and `alternateHdrHeadroom` is hardcoded to be 3 (6/2). So `alternateHdrHeadroom < baseHdrHeadroom` is always false. This seems incorrect.

Reply:
Changed this to swap base/alt headroom if base_rendition_is_hdr.